Kwenta/kwenta
# Enable synth redeem function on v2

## Description
1. Upgrade to the latest Synthetix packages: v2.73.1
2. Show a  `Redeem Deprecated Synths` button under the `Swap` button on the `exchange` page if the users have deprecated synths in their wallet

## Related issue
#1016 

## Motivation and Context
Enable the redemption function on v2

## How Has This Been Tested?
1. Connect a wallet with deprecated synths on the exchange page. See the `Redeem Deprecated Synths` button
2. Click on the button to see the `RedeemTxModal` which shows all the deprecated synths balance in the wallet
3. Confirm on the wallet and the transaction completed
https://kovan.etherscan.io/tx/0xeacfe2c7c8694628196fa5e39b0c09fe53e9cacafab45bb1ff1f45607f9e6da7

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/174130744-bc7e5bb4-d97a-40a8-80ec-fbc5713bbcf3.png)

Comparison between v1 table and v2 modal
![image](https://user-images.githubusercontent.com/4819006/174130190-52b0ac23-f948-4767-beba-8568e789a499.png)
